data/management/commands/fetch_deployed_data.py

```diff
@@ -14,13 +14,19 @@ class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument('output_dir', nargs='?', type=str)
         parser.add_argument('filenames', nargs='+', type=str)
+        parser.add_argument('--repo_name', action='store',
+                            dest='repo_name', type=str)
+        parser.add_argument('--allow_failure', action='store_true',
```

Comment:
this should be `--allow-failure` .

I should have done that with `output_dir`, but it is a nargs so the name isnt so critical.